### PR TITLE
add role descript param and update method

### PR DESCRIPTION
**Pull request by fooying**
* Add 'descript' param to createRole,Sometimes, we need the describe of role
* add 'setRoleDescript' method

### CODE REVIEW

**Comment by robertpitt** [comment]
Would it not make more sense to have a wider range of optional parameters, that is more app specific.

For example, I have a seperate roles system on my OAuth API called RBAC NIST L2, this has Roles and Permissions and the model has been extended to support a contextual id for groups.

I have extended the roles base to take an optional object for role properties, take the following snip

```js
Roles.createRole("system", {
    role_id: 8,
    context : 22,
    description: "System Role"
});
```

This allows me to create another package that imply's Roles package and extends the `Roles` namespace.

TLDR;
Just add a secondary, abstract object that can be used for many things, not just "descript" :)

**Comment by alanning** [comment]
@robertpitt, nice idea.  In general, I'd like to make `roles` more extensible so people can build on it to do other things such as integration with AD, timed access, limited access, etc.  If you have any suggestions on how to better enable that, please shoot me an email and we can chat off-thread.

**Comment by fooying** [comment]
@robertpitt good idea

**Comment by alanning** [comment]
Closing in favor of @robertpitt's suggestion.
